grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -23,21 +23,16 @@ class Order implements Serializable {
 
     def beforeInsert() {
         createdBy = AuthService.currentUser
+        updatedBy = AuthService.currentUser
     }
 
     def beforeUpdate() {
         updatedBy = AuthService.currentUser
     }
 
-    def publishRefreshEvent() {
-        if (isPurchaseOrder) {
-            Holders.grailsApplication.mainContext.publishEvent(new RefreshOrderSummaryEvent(this))
-        }
-    }
-
-    def publishRefreshEvenBeforeDelete() {
+    def publishRefreshEvent(Boolean forceRefresh = false) {
```

Comment:
it should be named `isDelete` - the same, as the second parameter of the `RefreshOrderSummaryEvent`